auth/standard_role_manager.cc

```diff
@@ -250,24 +251,32 @@ future<> standard_role_manager::start() {
             }
             return make_ready_future<>();
         }).then([this] {
-            _stopped = auth::do_after_system_ready(_as, [this] () -> future<void>  {
-                if (legacy_mode(_qp)) {
-                    co_await _migration_manager.wait_for_schema_agreement(_qp.db().real_database(), db::timeout_clock::time_point::max(), &_as);
+            _stopped = auth::do_after_system_ready(_as, [this] () -> future<>  {
+                auto signal_super_user_created = defer([this] {
+                    _superuser_created.signal();
+                });
+                try {
+                    if (legacy_mode(_qp)) {
```

Comment:
Please split the whitespace change to a separate commit so it's easier to review. (I.e. the commit where you introduce `try` block, do not shift everything to the right; shift it in a follow-up commit that fixes whitespace)